src/models/user.ts

```diff
@@ -15,6 +15,9 @@ const User = mongoose.model(
     role: {
       type: String,
       default: "user",
+      // type: mongoose.Types.ObjectId,
```

Comment:
Please check the review @ceelogre left on #13 referring to using `Role` model to assign roles. This will play a great role as we allow admins to create new roles and setting permissions

Reply:
Ok, Here we can use data from role's table.